code/modules/mob/living/carbon/human/say.dm

```diff
@@ -21,10 +21,116 @@
+		message = replacetext(message,"i don't know","wot mate")
+		message = replacetext(message,"no","naw")
+		message = replacetext(message,"robust","chin")
+		message = replacetext(message," hi ","how what how")
+		message = replacetext(message,"hello","sup bruv")
+		message = replacetext(message,"kill","bang")
+		message = replacetext(message,"murder","bang")
+		message = replacetext(message,"windows","windies")
+		message = replacetext(message,"window","windy")
+		message = replacetext(message,"break","do")
+		message = replacetext(message,"your","yer")
+		message = replacetext(message,"security","coppers")
+	if (dna.check_mutation(SMILE))
+		//Time for a friendly game of SS13
+		message = replacetext(message,"stupid","smart")
+		message = replacetext(message,"retard","genius")
+		message = replacetext(message,"unrobust","robust")
+		message = replacetext(message,"dumb","smart")
+		message = replacetext(message,"awful","great")
+		message = replacetext(message,"gay",pick("nice","ok","alright"))
+		message = replacetext(message,"horrible","fun")
+		message = replacetext(message,"terrible","terribly fun")
+		message = replacetext(message,"terrifying","wonderful")
+		message = replacetext(message,"gross","cool")
+		message = replacetext(message,"disgusting","amazing")
+		message = replacetext(message,"loser","winner")
+		message = replacetext(message,"useless","useful")
+		message = replacetext(message,"oh god","cheese and crackers")
+		message = replacetext(message,"jesus","gee wiz")
+		message = replacetext(message,"weak","strong")
+		message = replacetext(message,"kill","hug")
+		message = replacetext(message,"murder","tease")
+		message = replacetext(message,"ugly","beautiful")
+		message = replacetext(message,"douchbag","nice guy")
+		message = replacetext(message,"whore","lady")
+		message = replacetext(message,"nerd","smart guy")
+		message = replacetext(message,"moron","fun person")
+		message = replacetext(message,"IT'S LOOSE","EVERYTHING IS FINE")
+		message = replacetext(message,"rape","hug fight")
+		message = replacetext(message,"idiot","genius")
+		message = replacetext(message,"fat","thin")
+		message = replacetext(message,"beer","water with ice")
+		message = replacetext(message,"drink","water")
+		message = replacetext(message,"feminist","empowered woman")
+		message = replacetext(message,"i hate you","you're mean")
+		message = replacetext(message,"nigger","african american")
+		message = replacetext(message,"jew","jewish")
+		message = replacetext(message,"shit","shiz")
+		message = replacetext(message,"crap","poo")
+		message = replacetext(message,"slut","tease")
+		message = replacetext(message,"ass","butt")
+		message = replacetext(message,"damn","dang")
+		message = replacetext(message,"fuck","")
+		message = replacetext(message,"penis","privates")
+		message = replacetext(message,"cunt","privates")
+		message = replacetext(message,"dick","jerk")
+		message = replacetext(message,"vagina","privates")

```

Comment:
This is just horrible, i told you about on_say thing and hooks, dont butcher say with if checks and walls of text please
Make it nice